test/mainnet-fork/Ethena.t.sol

```diff
@@ -95,10 +95,38 @@ contract MainnetControllerPrepareUSDeMintFailureTests is ForkTestBase {
         mainnetController.prepareUSDeMint(100);
     }
 
+    function test_prepareUSDeMint_rateLimitBoundary() external {
+        vm.startPrank(SPARK_PROXY);
+        rateLimits.setRateLimitData(
+            mainnetController.LIMIT_USDE_MINT(),
+            100e18,
```

Comment:
Use realistic numbers. This is USDC so should be `e6` size.

Reply:
Updated